packages/js/src/ui/types.ts

```diff
@@ -55,7 +55,7 @@ export type NovuProviderProps = BaseNovuProviderProps & {
 };
 
 export enum NotificationStatus {
-  UNREAD_READ = 'unreadRead',
+  UNREAD_READ = 'all',
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** that's not meaning all, because all is: "unread+read+archived+unarchived"

Reply:
Do you think it matters that much for the key? I also did not like `unreadRead` as the name here. Should we name it `Inbox` or `home` or sth?

Reply:
yes, I think it might confuse the users if we call it `all` as the localization key

Reply:
Changed it!

Reply:
you changed the name for the enum but not the localization key